services/integration/src/controllers/constitution.controllers/constitution.js

```diff
@@ -1,7 +1,16 @@
-const Constitution = require('../models/constitution.model');
-const { asyncHandler } = require('../utils');
+const Constitution = require('../../models/constitution.model');
+const { asyncHandler } = require('../../utils');
 
 const create = asyncHandler(async (req, res) => {
+
+  const actOfConstitution = new Constitution(req.body);
+  await Constitution.save(function (err) {
```

Comment:
If you are usign async/await and the asynchandler helper you should not be adding a callback function.

Reply:
very good , I modify this endpoint!

Reply:
i have changed endpoint